service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/ActionCMOControllerTest.java

```diff
@@ -126,9 +141,68 @@ void aboutToSubmitShouldReturnCaseManagementOrderWithActionAndSchedule() throws
         assertThat(caseData.getCaseManagementOrder().getSchedule()).isEqualTo(createSchedule(true));
     }
 
+    @Test
+    void submittedShouldSendNotificationsWhenIssuedOrderApproved() throws Exception {
+        CaseManagementOrder approvedCaseManagementOrder = getCaseManagementOrder(getOrderAction());
+
+        Map<String, Object> data = ImmutableMap.of(
+            "familyManCaseNumber", FAMILY_MAN_CASE_NUMBER,
+            "respondents1", createRespondents(),
+            "caseLocalAuthority", LOCAL_AUTHORITY_CODE,
+            "caseManagementOrder", approvedCaseManagementOrder);
+
+        CallbackRequest callbackRequest = buildSubmittedCallbackRequest(data);
+
+        makeSubmittedRequest(callbackRequest);
+
+        verify(notificationClient, times(1)).sendEmail(
+            eq(CMO_ORDER_ISSUED_CASE_LINK_NOTIFICATION_TEMPLATE), eq(LOCAL_AUTHORITY_EMAIL_ADDRESS),
+            eq(getExpectedCMOIssuedCaseLinkNotificationParameters()), eq("12345"));
+    }
+
+    @Test
+    void submittedShouldNotSendNotificationsWhenIssuedOrderNotApproved() throws Exception {
+        CaseManagementOrder caseManagementOrder = getCaseManagementOrder(OrderAction.builder().build());
+
+        Map<String, Object> data = ImmutableMap.of(
+            "familyManCaseNumber", FAMILY_MAN_CASE_NUMBER,
+            "respondents1", createRespondents(),
+            "caseLocalAuthority", LOCAL_AUTHORITY_CODE,
+            "orderAction", OrderAction.builder()
+                .type(ActionType.JUDGE_REQUESTED_CHANGE)
+                .nextHearingType(ISSUES_RESOLUTION_HEARING)
+                .build(),
+            "caseManagementOrder", caseManagementOrder);
+
+        CallbackRequest callbackRequest = buildSubmittedCallbackRequest(data);
+
+        makeSubmittedRequest(callbackRequest);
+
+        verify(notificationClient, times(0)).sendEmail(
```

Comment:
There's `never()` method which is an alias for `times(0)`, but feel free to resolve.